.gitignore

```diff
@@ -29,3 +29,4 @@ package-lock.json
 *.tsbuildinfo
 packages/webapp/webpack-stats.json
 nbdime/labextension/
+/src/
```

Comment:
running `pip install -e git+https://github.com/jupyter/nbdime#egg=nbdime` adds this folder, but probably shouldn't be included in VCS — lets git ignore it